### PR TITLE
Final check of IDA content

### PR DESCRIPTION
**Pull request by CitMC**
This PR includes minor edits of the IDA content updates.

### Change scope

The changes in this Pull Request include (please tick all that apply):

- [x] Small language/grammar fixes
- [ ] Small content fixes 
- [ ] Addition of new content
- [ ] Sample code/command updates
- [ ] Platform fixes
- [ ] Other

### CODE REVIEW

File: academy/4-my-own-chain/server-side.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ tag: deep-dive
 
 # CosmJS on a Backend Script for Game Indexing
 
-Now that your blockchain is complete, you can think about additional data and services that would add value without increasing cost or complexity on chain.
+Now that your blockchain is complete, you can think about additional data and services that would add value without increasing cost or complexity on-chain.
```
Comment (xavierlepretre) [inline academy/4-my-own-chain/server-side.md:10]: I think here it does not take the `-`.

File: academy/ibc/clients.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ The Tendermint client `ConsensusState` tracks the timestamp of the block being c
 
 <HighlightBox type="tip">
 
-The next validator set is used for verifying subsequent submitted headers or updates to the counterparty `ConsensusState`. See the following part on _Updating clients_ for more information about what happens when a validator set changes between blocks.
+The next validator set is used for verifying subsequent submitted headers or updates to the counterparty `ConsensusState`. See the following part on [Updating clients](https://interchainacademy.cosmos.network/academy/ibc/clients.html#updating-a-client) for more information about what happens when a validator set changes between blocks.
```
Comment (xavierlepretre) [inline academy/ibc/clients.md:169]: Why not use `(#updating-a-client)`?

**Review by xavierlepretre — APPROVED** [review]

**Review by coldice — APPROVED** [review]

File: academy/ibc/clients.md (hunk 2 above)
Comment (coldice) [reply academy/ibc/clients.md]: note that the link should be put as `./#updating-a-client` per https://v1.vuepress.vuejs.org/guide/markdown.html#:~:text=%5Bfoo%20heading%5D(./%23heading)

Comment (coldice) [reply academy/ibc/clients.md]: note that this actually doesn't work 👎 replaced with
`./clients.md#updating-a-client`

**Pull request merged**